shared/actions/search.js

```diff
@@ -149,7 +149,7 @@ function search (term: string, maybePlatform: ?SearchPlatforms) : TypedAsyncActi
     const service = {
       'Coinbase': 'coinbase',
       'Facebook': 'facebook',
-      'Github': 'github',
+      'GitHub': 'github',
       'Hackernews': 'hackernews',
```

Comment:
Whilst you're at it, you could fix "Hackernews" -> "Hacker News" and maybe "Pgp" -> "PGP" (here and elsewhere).